src/index.js

```diff
@@ -1,18 +1,34 @@
 import React, { useState, useEffect } from 'react'
 import { isAuthenticated } from '@babbage/sdk'
 import Prompt from './components/Prompt'
+import UnsupportedBrowser from './components/UnsupportedBrowser'
+import BravePrompt from './components/BravePrompt'
 import Theme from './components/Theme'
 
 const checkStatus = async () => {
   try {
     const authenticated = await isAuthenticated(undefined, false)
     if (!authenticated) {
-      return false
+      return {
+        authenticated: false,
+        supportedBrowser: true
+      }
     }
   } catch (e) {
-    return false
+    let supportedBrowser = false
+    if (e.message === 'Failed to fetch') {
+      // Babbage MetaNet Client is not active
+      supportedBrowser = true
+    }
+    return {
+      authenticated: false,
+      supportedBrowser
+    }
+  }
+  return {
+    authenticated: false,
```

Comment:
```suggestion
    authenticated: true,
```
It should be true :)

Reply:
True dat